peripherals/zl30260.py

```diff
@@ -66,7 +66,7 @@
 #                                  S O F T W A R E                                                 #
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
 
-class zl30260Driver:
+class ZL30260Driver:
```

Comment:
The other definitions at the top of this file need to also have the `ZL` capitalized.